src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -108,7 +109,7 @@ public static function collectDeprecations($outputFile)
                 return $ErrorHandler::handleError($type, $msg, $file, $line, $context);
             }
 
-            $deprecations[] = [error_reporting(), $msg, $file];
+            $deprecations[] = [error_reporting(), $msg, $file, serialize(debug_backtrace())];
```

Comment:
that cannot work I think: serializing the backtrace will break when anything is found there that fails to be serialized
@greg0ire maybe we need your help to restore this feature.

Reply:
I'm agree :) I asked for some clarifications on #29211 to @greg0ire. This is used to determine if the deprecation is direct/indirect when come from vendors. But maybe not be able to determine that in the edge case of runInSeparateProcess when trace can't be serialized is acceptable ?

Reply:
I want to help but will be AFK for 4 more days.

Reply:
I changed the appoach